### ISSUE TITLE
BeanIOConfigurationException from a repeating segment

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#34]
```
In release 2.0.0.M1, when a mapping file is loaded, a repeating segment may 
cause a BeanIOConfigurationException when followed by a field or another 
segment in the same record.

```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 19 May 2012 at 2:58

**Comment by GoogleCodeExporter** [comment]
```
Fixed for release 2.0.0.
```

Original comment by `kevin.s...@gmail.com` on 20 May 2012 at 2:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 20 May 2012 at 9:39
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Found a few more occurrences that may cause a similar error.
```

Original comment by `kevin.s...@gmail.com` on 1 Jul 2012 at 7:44
- Changed state: **Started**
- Added labels: **Milestone-2.0.1**
- Removed labels: **Milestone-2.0.0**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 2.0.1.
```

Original comment by `kevin.s...@gmail.com` on 1 Jul 2012 at 9:00
- Changed state: **Fixed**
